.github/workflows/build-tarball.yml

```diff
@@ -1,6 +1,10 @@
 name: Build from tarball
 
 on:
+  workflow_run:
+    workflows: ["lint-addon-docs", "lint-cpp", "lint-js-and-md", "lint-py", "lint-yaml", "lint-sh", "lint-codeowners", "lint-pr-url"]
+    types:
+      - completed
```

Comment:
This doesn't work. It doesn't prevent the workflow from running on the pull_request event.
Also, the `workflows` array is not a list of dependencies that must all run before. Just one of them is enough. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run

Reply:
Its a draft PR, I am adding the options `needs` . But, trying to understand how the workflows are running, how to optimise the order. 

To keep the actions to run, not adding the draft label.

Reply:
<img width="400" alt="Screenshot 2023-11-04 at 7 40 49 pm" src="https://github.com/nodejs/node/assets/26359740/a9a4ef56-8094-4cf5-bda6-d5f989211180">

Something like this will be the target.

Reply:
Why would we want that? At least to me it would be very annoying to not have immediate feedback on everything. The commit message linting is not a prerequisite for the other lint jobs to run and succeed.

Reply:
If the lint message is invalid, we really don't want to lint the other files right? 

If the commit message needs some correction, then the developer will push the changes again. So, to avoid the multiple runs, thought put it as the first prerequisite

Reply:
<img width="600" alt="Screenshot 2023-11-04 at 7 59 26 pm" src="https://github.com/nodejs/node/assets/26359740/14a9b2d0-fdf9-4249-bb44-d817a9e63704">

Something like this.

Reply:
I do not agree with that. If the commit message is wrong. The developer would still want to know if there are other issues. Otherwise you lose time because you have to fix the commit message before getting feedback on the other things.

Reply:
Okay. We can discard the approach. No worries. 

I got comment on the issue which I raised for this change to make this way.

So, thought to experiment that